

const pushData = (req, res) => {
    try {

    } catch (error) {
        console.log(error);
        throw error;
    }
}